whois-nrtm4/src/main/java/net/ripe/db/nrtm4/util/SnapshotUpdateWindow.java

```diff
@@ -0,0 +1,39 @@
+package net.ripe.db.nrtm4.util;
+
+import net.ripe.db.nrtm4.persist.SnapshotFile;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
```

Comment:
Don't use `java.util.Date` but use `java.time` instead. There's a `net.ripe.db.whois.common.domain.Timestamp` class in whois-commons that wraps a SQL timestamp.

For more background, see e.g.
https://programminghints.com/2017/05/still-using-java-util-date-dont/

Reply:
removed this 'sketch' from the branch